pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -67,7 +67,7 @@ def data_for_grouping():
 
 class TestDecimalArray(base.ExtensionTests):
     def _get_expected_exception(
-        self, op_name: str, obj, other
+        self, op_name: str, obj, other, request
```

Comment:
The `request` in this file don't look like they are used?

Reply:
Fixed by using LSP parameters elsewhere.